crates/red_knot_python_semantic/resources/mdtest/annotations/callable.md

```diff
@@ -209,5 +209,3 @@ def _(c: Callable[[int], int]):
     # error: [unresolved-attribute] "Type `(int, /) -> int` has no attribute `__call__`"
     reveal_type(c.__call__)  # revealed: Unknown
 ```
-
-[gradual form]: https://typing.readthedocs.io/en/latest/spec/glossary.html#term-gradual-form
```

Comment:
I think this should still be present as there are references to it in the document.